src/client.ts

```diff
@@ -75,6 +75,8 @@ export interface ClientOptions {
   logger?: any;
   optOutOfWebrtcStatsTelemetry?: boolean;
   allowIPv6?: boolean;
+  parentAppName?: string;
```

Comment:
Do you think it makes sense to just call this `appName` and `appVersion`?

Reply:
Yeah, I think that is better since the consuming app doesn't really care that it is the "parent".